history/cultures/wc_azerothian.txt

```diff
@@ -0,0 +1,51 @@
+#Human group
+1.1.1 = {
+	discover_innovation = innovation_motte
+	discover_innovation = innovation_barracks
+	#
+	discover_innovation = innovation_development_01
+}
+
+270.1.1 = {
+	discover_innovation = innovation_catapult
+	discover_innovation = innovation_mustering_grounds
+	discover_innovation = innovation_quilted_armor
+	#
+	discover_innovation = innovation_currency_01
+	discover_innovation = innovation_casus_belli
+}
+
+400.1.1 = {
+	discover_innovation = innovation_bannus
+	#
+	discover_innovation = innovation_gavelkind
+	discover_innovation = innovation_ledger
+	#
+	discover_innovation = innovation_elite_ranks
+	join_era = culture_era_early_medieval
```

Comment:
It doesn't work. Humans are in the Tribal Era.

Reply:
The bug is still present.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/101607312-e4adf380-3a1d-11eb-9f69-b2ab8d991560.png)

</details>